trigger.js

```diff
@@ -215,39 +224,76 @@ class TriggerHappy {
     _doCaptureTriggers(token, scene, update) {
         // Get all trigger tokens in scene
         let targets = this._getTokensFromTriggers(canvas.tokens.placeables, this.triggers, 'capture');
-        if (!targets) return true;
+        if (!targets) return;
 
         const finalX = update.x || token.x;
         const finalY = update.y || token.y;
         // need to calculate this by hand since token is just token data
-        const tw = token.width * canvas.scene.data.grid / 2;
-        const th = token.height * canvas.scene.data.grid / 2;
-        const motion = new Ray({x: token.x + tw, y: token.y + th}, {x: finalX + tw, y: finalY + th});
+        const tkw = token.width * canvas.scene.data.grid / 2;
+        const tkh = token.height * canvas.scene.data.grid / 2;

```

Comment:
Ususally a simple tw/th like this is ok, but when it becomes hard to know what it is from the name, like 'tkw' and 'tgw' below, then it's best to use fuller names, like `tokenWidth` and `targetWidth`